examples/gno.land/p/demo/json/eisel_lemire.gno

```diff
@@ -0,0 +1,930 @@
+package json
+
+// This file implements the Eisel-Lemire ParseFloat algorithm, published in
+// 2020 and discussed extensively at
+// https://nigeltao.github.io/blog/2020/eisel-lemire.html
+//
+// The original C++ implementation is at
+// https://github.com/lemire/fast_double_parser/blob/644bef4306059d3be01a04e77d3cc84b379c596f/include/fast_double_parser.h#L840
+//
+// This Go re-implementation closely follows the C re-implementation at
+// https://github.com/google/wuffs/blob/ba3818cb6b473a2ed0b38ecfc07dbbd3a97e8ae7/internal/cgen/base/floatconv-submodule-code.c#L990
+//
+// Additional testing (on over several million test strings) is done by
+// https://github.com/nigeltao/parse-number-fxx-test-data/blob/5280dcfccf6d0b02a65ae282dad0b6d9de50e039/script/test-go-strconv.go
+
+import (
+	"math"
+	"math/bits"
```

Comment:
right now CI is failing because `math/bits` is not in the whitelist.

We have `math/bits` in the stdlibs, but not in the whitelist. Strange!!!
Please add it to the `stdlibWhitelist`

Reply:
nice catch! now I added in whitelist